### ISSUE TITLE
Fix EasyCLA build process error

### ISSUE THREAD

**Issue by brianorwhatever** [trustoverip/ctwg-main-glossary#11]
It looks like the github pages build fails with EasyCLA problems 😞

I'm not sure how to resolve this issue

**Comment by kordwarshuis** [comment]
Does this info help?
https://wiki.trustoverip.org/display/HOME/EasyCLA+onboarding
(via @darrellodonnell )

**Comment by m00sey** [comment]
I'll take a look.

**Comment by henkvancann** [comment]
> Does this info help? https://wiki.trustoverip.org/display/HOME/EasyCLA+onboarding (via @darrellodonnell )
Yes, it did for me. Had to sign a document and then all signs green.

**Comment by m00sey** [comment]
I've added a branch protection rule allowing force pushes to the gh-pages branch, so the workflow should be fine now.

**Comment by brianorwhatever** [comment]
@m00sey excellent, let's give it a try by merging in #9

**Comment by darrellodonnell** [comment]
> I've added a branch protection rule allowing force pushes to the gh-pages branch, so the workflow should be fine now.

Have we documented what is required here (i.e. how to reproduce the proper setup)?

**Comment by brianorwhatever** [comment]
Looks like this worked and is a valid path forward to resolve the EasyCLA / gh-pages issues we've encountered. Just needs to be documented so it can be reproduced the next time this crops up

**Comment by m00sey** [comment]
needs to be part of new repository set up, yes.
